src/win/util.c

```diff
@@ -79,17 +79,16 @@ static uint64_t hrtime_frequency_ = 0;
  */
 void uv__util_init(void) {
   LARGE_INTEGER perf_frequency;
-
   /* Initialize process title access mutex. */
   InitializeCriticalSection(&process_title_lock);
-
   /* Retrieve high-resolution timer frequency
    * and precompute its reciprocal.
    */
   if (QueryPerformanceFrequency(&perf_frequency)) {
     hrtime_frequency_ = perf_frequency.QuadPart;
   } else {
     uv_fatal_error(GetLastError(), "QueryPerformanceFrequency");
+    DeleteCriticalSection(&process_title_lock);
```

Comment:
Unreachable code. The uv_fatal_error() on the preceding line is what the name says: fatal. It doesn't return.

Reply:
this should free it first before go to uv_fatal_error(), i will update the PR

Reply:
It doesn't matter. uv_fatal_error() aborts and then everything gets cleaned up automatically by the operating system.

Reply:
Hmm interesting.. But if that is already handled by OS, we don't need to `DeleteCriticalSection` and this change is also not relevant again (?)

Reply:
Correct, it's pointless and possibly confusing to future readers of the code ("why is it trying to clean up when we're exiting after right away?")